### ISSUE TITLE
Latest release (0.8.0) fails version checks in cmake and pkg-confg

### ISSUE THREAD

**Issue by phire** [Cyan4973/xxHash#524]
Tying to do a minimum version check in cmake fails when 0.8.0 is installed. 

For example: `pkg_check_modules(XXHASH libxxhash>=0.7.3 REQUIRED)` generates an error of:

```[build] -- Checking for module 'libxxhash>=0.7.3'
[build] --   Package dependency requirement 'libxxhash >= 0.7.3' could not be satisfied.
[build] Package 'libxxhash' has version '', required version is '>= 0.7.3'
[build] -- Configuring incomplete, errors occurred!
```

This is because `pkg-config libxxhash --modversion` returns an empty string.

Looks like this was fixed in b4dbf5fefc37b8a5f80b3bbc6b8ff639cf5f0ec2, just a few hours after release, but there have been no follow up releases since then so the stable actual release is still broken. Some distros (like [debian](https://sources.debian.org/src/xxhash/0.8.0-2/debian/patches/upstream-b4dbf5fefc37b8a-fix-empty-version-in-pc-file/)) have patched this, but others (like [arch linux](https://github.com/archlinux/svntogit-community/blob/packages/xxhash/trunk/PKGBUILD)) haven't.

**Comment by ThomasWaldmann** [comment]
Can confirm, I also stumbled over some dist provided xxhash that has no version in the package-config.

**Comment by Cyan4973** [comment]
fix will be present in `v0.8.1`
